server/athenian/api/openapi/openapi.yaml

```diff
@@ -1056,3 +1085,163 @@ components:
       - date_from
       - date_to
       type: object
+    FilterPullRequestsRequest:
+      example:
+        account: 1
+        date_from: 2000-01-23
+        date_to: 2020-05-23
+        in:
+        - '{1}'
+        stages:
+        - wip
+      properties:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: PRs must be updated later than or equal to this date.
+          format: date
+          type: string
+        date_to:
+          description: PRs must be updated earlier than or equal to this date.
+          format: date
+          type: string
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        stages:
+          items:
+            $ref: '#/components/schemas/PullRequestPipelineStage'
+          type: array
+        with:
+          $ref: '#/components/schemas/PullRequestWith'
+      required:
+      - account
+      - date_from
+      - date_to
+      - stages
+      type: object
+    PullRequestPipelineStage:
+      description: One of the four possible PR states of the modelled lifecycle.
+      enum:
+      - wip
+      - review
```

Comment:
review, merge, release are verbs; I'd use adjectives instead, which are more meaningful to describe PR states:
- reviewed,
- merged,
- released.

Reply:
This is the same pitfall Waren fell into and led to metrics confusion: they are processes, not events. I can make them "reviewing", "merging", "releasing".

Reply:
yes... I reviewed again our docs... and you're right.